### PR TITLE
Get Inkscape path from Windows registry

### PR DESCRIPTION
**Pull request by juhasch**
Otherwise Inkscape needs to be manually added to the path variable.

### CODE REVIEW

File: nbconvert/preprocessors/svg2pdf.py [hunk 1]
```diff
@@ -20,6 +20,12 @@
 
 INKSCAPE_APP = '/Applications/Inkscape.app/Contents/Resources/bin/inkscape'
 
+if sys.platform == "win32":
+    if sys.version_info < (2, 99):
```
Comment (minrk) [inline nbconvert/preprocessors/svg2pdf.py]: Probably best to use `try import/except import` instead of targeting a specific version

**Comment by minrk** [comment]
Minor note in how to handle version checking, but otherwise +1

**Comment by juhasch** [comment]
Now using try/except to import winreg module

**Comment by meeseeksmachine** [comment]
This pull request has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/inkscape-not-linking-for-pdf-export/4292/1

**Pull request merged**